crates/store/re_log_encoding/benches/msg_encode_benchmark.rs

```diff
@@ -32,7 +32,7 @@ criterion_group!(
 criterion_main!(benches);
 
 fn encode_log_msgs(messages: &[LogMsg]) -> Vec<u8> {
-    let encoding_options = re_log_encoding::EncodingOptions::COMPRESSED;
+    let encoding_options = re_log_encoding::EncodingOptions::MSGPACK_COMPRESSED;
```

Comment:
Wouldn't it make more sense to benchmark the new "standard" method, i.e. `PROTOBUF_COMPRESSED`? That way we can test the old vs new performance

Reply:
Still, would be interesting to see how `PROTOBUF_COMPRESSED` and `MSGPACK_COMPRESSED` compare to each other in encode/decode speed